Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -3774,6 +3774,8 @@ script:
       name: ip_netmask
     - description: The IP range of the new address IP. For example, 10.10.10.0-10.10.10.255
       name: ip_range
+    - description: The IP wildcard of the new address. For example, 10.20.1.0/0.0.248.255
```

Comment:
where does 0.0.248.255 come from? Doesnt look like a valid subnet to me. are the 0s wildcard numbers here?

Reply:
its taken directly from their documentation
<img width="903" alt="image" src="https://user-images.githubusercontent.com/53861351/191678676-566f86ae-5d4e-404b-b29b-06fb203e16ba.png">